documentation/website/docs/installation.md

```diff
@@ -71,6 +72,21 @@ $ source ~/.bashrc
 ```
 You should be able to open a new shell and run `pyre -h` now, confirming `pyre` was set-up correctly. Any changes made to the Pyre Python client code should be immediately observable the next time you invoke `pyre`
 
+#### Testing changes for Plugin Development
+VSCode will not pick up your shell aliases, so the alias step in the previous section will not work if you're doing VSCode Plugin development. To get around this, instead of creating an alias, we can create an executable script called `pyre` and place it in a directory in our `PATH`:
+
+```bash
+#!/bin/bash
+export PYTHONPATH="/path/to/pyre-check:$PYTHONPATH"
```

Comment:
Looks like you forgot to delete this line

Reply:
Yeah sorry about that